src/utils/words.ts

```diff
@@ -6,6 +6,7 @@ import type { IsDigit } from './characters/numbers.js'
 import type { IsSpecial } from './characters/special.js'
 import type { IsStringLiteral } from '../internal/literals.js'
 
+// prettier-ignore
```

Comment:
Didn't want the type to turn into indentation hell, open to suggestions for a better solution